app/views/firms/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="l-constrained">
+  <div class="l-firm">
+    <div class="l-firm__main">
+      <%= heading_tag(t('firms.show.heading'), level: 1, class: 'l-firm__heading') %>
+      <p>
+        <%= link_to :back do %>
```

Comment:
I'd say no — I had imagined we'd do that later but I can't think of any particularly good reason not to do it now, since we have the data